src/templates/auth_manifests.yaml.j2

```diff
@@ -4,6 +4,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: leader-election-role
```

Comment:
we should avoid generic names like this, let's keep the names the same as the output from building upstream manifests, for example here it would be `seldon-leader-election-role`